src/Components/CollectorProfile/CollectorProfileArtistsAdd.tsx

```diff
@@ -82,7 +79,6 @@ export const CollectorProfileArtistsAdd: FC<React.PropsWithChildren<CollectorPro
           category: "COLLECTED_BEFORE" as UserInterestCategory,
           interestId,
           interestType: "ARTIST" as UserInterestInterestType,
-          private: true,
```

Comment:
Per default, the user interest isn't private, so removing the argument is enough to share the artist's interest with galleries.